themes/ranked-choice-voting-for-texas-5dabb2094445ea66232bca7e/pages_show_volunteer_signup.html

```diff
@@ -48,16 +48,43 @@ <h4>Will you volunteer?</h4>
 
       <div class="form-errors">{% error_messages_for volunteer_signup %}</div>
 
+      {% comment %}
+        Liquid doesn't allow you to initialize arrays or edit strings, but it
+        does have this weird capture feature that's annoyingly sensitive to
+        spaces. We're making due with what we've got.
+      {% endcomment %}
+
+      {% capture categories %}{% for volunteer_type in page.volunteer_signup.volunteer_types %}{% assign type_info = volunteer_type.name | split: " - " %}{% if type_info.size > 1 %}{{ type_info[0] }},{% endif %}{% endfor %}{% endcapture %}
+
+      {% assign categories_list = categories | split: "," %}
+      {% assign unique_categories = categories_list | uniq %}
+
+      {% comment %}
+        Group checkboxes by volunteer category.
+      {% endcomment %}
+
       <div class="volunteer-signup-types">
-        {% for volunteer_type in page.volunteer_signup.volunteer_types %}
-        <div class="row-fluid">
-          <div class="span12"><label class="checkbox">
-            {% check_box_tag 'volunteer_signup[volunteer_type_ids][]', volunteer_type.id, collection:volunteer_signup.volunteer_type_ids, member:volunteer_type.id, class:"checkbox" %}
-            {{ volunteer_type.name }}
-          </label></div>
+      {% for category in unique_categories %}
+        {% if category != " " %}
+        <div class="padtop">
+          {{ category }}
         </div>
-        {% endfor %}
+          {% for volunteer_type in page.volunteer_signup.volunteer_types %}
+            {% if volunteer_type.name contains category %}
+          <div class="row-fluid">
+            <div class="span12"><label class="checkbox">
+              {% assign type_info = volunteer_type.name | split: " - " %}
```

Comment:
Nit: What about `type_info` -> `volunteer_category`?

Reply:
Nit just means a suggested change but can ignore it if you want. https://google.github.io/eng-practices/review/reviewer/standard.html

Reply:
Oh! Yes, both your naming recommendations sound much more descriptive. I'll make the changes, thanks!